src/JSON.jl

```diff
@@ -88,7 +88,7 @@ function _print(io::IO, state::State, s::AbstractString)
     Base.print(io, '"')
 end
 
-function _print(io::IO, state::State, s::Symbol)
+function _print{S <: Union{Symbol,Enum}}(io::IO, state::State, s::S)
```

Comment:
I'm saying that it can be `function _print(io::IO, state::State, s::Union{Symbol,Enum})`. I don't have an opinion on if supporting `Enum` this way is necessary or how/if it should be supported.